src/index.js

```diff
@@ -13,11 +13,32 @@ import { isValid as isValidEin, mask as maskEin } from 'ein-validator';
 import { isValid as isValidItin, mask as maskItin } from 'itin-validator';
 import { isValid as isValidSsn, mask as maskSsn } from 'ssn-validator';
 
+/**
+ * Excludes repeated numbers as TIN e.g. 111111111.
+ */
+
+const repeatedNumbers = Array.from({ length: 10 }, (_, i) => i).map(current => String(current).repeat(9));
+
+/**
+ * Excludes ascending and descending sequence as TIN e.g. 123456789.
+ */
+
+const sequence = Array.from({ length: 10 }, (_, i) => i).reduce((acc, current) => acc + current, '').repeat(2);
+const reverseSequence = sequence.split('').reverse().join('');
+
 /**
  * Export `isValid` function.
  */
 
 export function isValid(value) {
+  if (repeatedNumbers.indexOf(value.replace(/\D/g, '')) !== -1) {
+    return false;
+  }
+
+  if (sequence.includes(value) || reverseSequence.includes(value)) {
```

Comment:
Hmm, missing `value.replace(/\D/g, '')`? We should this at the beginning of the function